dev/com.ibm.ws.beanvalidation.v11_fat/test-applications/EJBModule1EJB.jar/resources/META-INF/constraints-house_EE9.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<constraint-mappings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://jboss.org/xml/ns/javax/validation/mapping validation-mapping-1.0.xsd"
+  xmlns="http://jboss.org/xml/ns/javax/validation/mapping">
+  <default-package>beanvalidation.ejbmodule.ejb</default-package>
+
+  <bean class="AValidationXMLTestBean" ignore-annotations="true">
+    <field name="builder">
+      <constraint annotation="jakarta.validation.constraints.NotNull" />
```

Comment:
Is the transformer tool supposed to work with arbitrary xml file names?  These files could be named anything in theory. Otherwise we'd be adding FAT specific scanning to the tool. We could scan all XML for `constraint-mappings`, or add "smarts" to read the `validation.xml` and figured out what constraint mapping files it points to. I wasn't sure if that was in the scope of the tooling. @tbitonti Your thoughts?

Reply:
Is `constraints-house.xml` an arbitrary name or a spec-defined name? The scanner tool is handing spec-defined XML files like web.xml and beans.xml, so it should also handle validation.xml's too

Reply:
I've talked with Tom and opened issue #12431 .